dockers/broad/samtools/docker_versions.tsv

```diff
@@ -0,0 +1,2 @@
+DOCKER_VERSION
+us.gcr.io/broad-gotc-prod/samtools:1.0.0-1624651616
```

Comment:
Same comment that I think this should be/include the samtools version (1.11) primarily - and then presumably the docker version and timestamp?  Maybe discuss.